warehouse/forklift/legacy.py

```diff
@@ -1059,7 +1059,7 @@ def file_upload(request):
 
 def _legacy_purge(status, *args, **kwargs):
     if status:
-        requests.post(*args, **kwargs)
+        requests.get(*args, **kwargs)
```

Comment:
I think this needs to remain a `POST`? I'm not familiar with the legacy API but I'm pretty sure this won't correctly send the `:action` and `project` via the `data` kwarg.

Reply:
Oops, beat me to it. 👍

Reply:
confirmed, i did a dumb :)